automatic/crystaldiskinfo.install/update.ps1

```diff
@@ -1,19 +1,20 @@
 Import-Module Chocolatey-AU
 
 $releases = 'https://crystalmark.info/redirect.php?product=CrystalDiskInfoInstaller'
-$feed = 'https://osdn.net/projects/crystaldiskinfo/releases/rss'
+$downloads = 'https://sourceforge.net/projects/crystaldiskinfo/files/'
```

Comment:
Please use the SourceForge RSS feed for downloads. It is more stable and less trouble when the site changes.

Example: smartmontools package.

Reply:
Fixed